makelib/variables.mk

```diff
@@ -6,7 +6,9 @@ override BUILDDIR := $(abspath $(BUILDDIR))
 
 STAMPSDIR := $(BUILDDIR)/stamps
 TOOLSDIR := $(BUILDDIR)/tools
-BINDIR := $(BUILDDIR)/bin
+TARGETDIR := $(BUILDDIR)/target
```

Comment:
I was wondering if the target directory can contain an arch in its name, something like `$(BUILDDIR)/target-$(RKT_ACI_ARCH)` or `$(BUILDDIR)/target-$(GOARCH)`.